hotel-management/src/pages/HomePage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+
+import { Button } from '@/components/Button';
+import './index.css';
+import { RoomCard } from '@/components/RoomCard/RoomCard';
+import { Room } from '@/types';
+
+export const HomePage = () => {
+  const testRooms: Room[] = [
+    {
+      id: '1',
+      name: '101',
+      roomType: 'single',
+      price: 150.0,
+      isAvailable: true,
+    },
+    {
+      id: '2',
+      name: '102',
+      roomType: 'double',
+      price: 200.0,
+      isAvailable: false,
+    },
+    {
+      id: '3',
+      name: '103',
+      roomType: 'double',
+      price: 300.0,
+      isAvailable: false,
+    },
+    {
+      id: '4',
+      name: '104',
+      roomType: 'double',
+      price: 300.0,
+      isAvailable: false,
+    },
+  ];
+
+  return (
+    <main className="container homepage-container">
+      <div className="btn-wrapper-row">
+        <Link to="/">
+          <Button color="tertiary" variant="outline" size="md">
+            Add Room
+          </Button>
+        </Link>
+      </div>
+      <div>
+        <ul className="list-room">
+          {testRooms.map(({ id, name, price, isAvailable }) => (
+            <RoomCard
+              key={id}
+              roomId={id}
+              name={name}
+              price={price}
+              isAvailable={isAvailable}
+              src=""
+              onDelete={() => {}}
+              onEdit={() => {}}
+            ></RoomCard>
```

Comment:
For any empty component make sure u use this syntax `<RoomCard />` instead of `<RoomCard><RoomCard />`